StantUiIosLibrary/Classes/Bars/SearchBar/DefaultSearchBar.swift

```diff
@@ -34,7 +34,7 @@ public class DefaultSearchBar: UIView {
         searchBar = UISearchBar(frame: CGRect(x: 0, y: 0,
                                               width: self.frame.width - 12,
                                               height: self.frame.height))
-        
+        searchBar?.tag = 4
```

Comment:
Why is this tag setted as 4?

Reply:
It was tagged as 4 because when I put as 1 (as expected) it affected on the its superviews tags. But, with your question, I decided to changed it. I put as 1 and changed the way that I used it, thanks.